docs/output.md

```diff
@@ -6,554 +6,671 @@ This document describes the output produced by the pipeline. Most of the plots a
 
-- `[SAMPLE].recal.table`
-  - Recalibration table associated to the `duplicates-marked BAM` file.
+These files are intermediate and by default not kept in the final files delivered to users. Set `--save_bam_mapped` to enable publishing.
 
-#### GATK ApplyBQSR
+<details markdown="1">
+<summary>Output files for all mappers and samples</summary>
 
-[GATK ApplyBQSR](https://gatk.broadinstitute.org/hc/en-us/articles/360042476852-ApplyBQSR) recalibrates the base qualities of the input reads based on the recalibration table produced by the [GATK BaseRecalibrator](#gatk-baserecalibrator) tool.
+**Output directory: `{outdir}/preprocessing/<sample>/mapped`**
 
-This directory is the location for the final `recalibrated BAM` files.
-`Recalibrated BAM` files are usually 2-3 times larger than the `duplicates-marked BAM` files.
-To re-generate `recalibrated BAM` file you have to apply the recalibration table delivered to the `DuplicatesMarked\` folder either using `Sarek` ( [`--step recalibrate`](usage.md#step-recalibrate) ) , or doing this recalibration yourself.
+- if `--save_bam_mapped`: `<sample>.bam` and `<sample>.bam.bai`
+  - BAM file and index
 
-For all samples:
+</details>
 
-**Output directory: `results/Preprocessing/[SAMPLE]/Recalibrated`**
+### Mark Duplicates
 
-- `[SAMPLE].recal.bam` and `[SAMPLE].recal.bam.bai`
-  - `BAM` file and index
+During duplicate marking, read pairs that are likely to have originated from duplicates of the same original DNA fragments through some artificial processes are identified. These are considered to be non-independent observations, so all but a single read pair within each set of duplicates are marked, causing the marked pairs to be ignored by default during the variant discovery process.
 
 For further reading and documentation see the [data pre-processing for variant discovery from the GATK best practices](https://gatk.broadinstitute.org/hc/en-us/articles/360035535912-Data-pre-processing-for-variant-discovery).
 
-### CSV files
-
-The `CSV` files are auto-generated and can be used by `Sarek` for further processing and/or variant calling.
-
-For further reading and documentation see the [`--input`](usage.md#--input) section in the usage documentation.
+#### GATK MarkDuplicates (Spark)
 
-For all samples:
+By default, Sarek will use [GATK MarkDuplicates](https://gatk.broadinstitute.org/hc/en-us/articles/5358880192027-MarkDuplicates-Picard-).
 
-**Output directory: `results/Preprocessing/TSV`**
+Specify `--use_gatk_spark markduplicates` to use [`GATK MarkDuplicatesSpark`](https://gatk.broadinstitute.org/hc/en-us/articles/5358833264411-MarkDuplicatesSpark) instead, the corresponding spark implementation. The resulting files are converted to CRAM with either [samtools](https://www.htslib.org/doc/samtools.html), when GATK MarkDuplicates is used, or, implicitly, by GATK MarkDuplicatesSpark.
```

Comment:
I find this sentence a bit confusing. Consider changing this maybe to:

To use the corresponding spark implementation instead, please specify ...